src/features/payments/routers/notices.ts

```diff
@@ -87,3 +89,21 @@ addNoticesHandler("get", "/paids/:eventId/pdf", (req, res) => {
     )
   );
 });
+
+addNoticesHandler("post", "/paids/:eventId/disable", (req, res) => {
+  pipe(
+    req.params.eventId,
+    O.fromNullable,
+    O.fold(
+      () => res.sendStatus(400),
+      eventId =>
+        pipe(
+          O.fromNullable(NoticesDB.removeUserNotice(eventId)),
+          O.fold(
+            () => res.sendStatus(hideReceiptResponseCode),
+            () => res.sendStatus(hideReceiptResponseCode)
+          )
+        )
+    )
+  );
+});
```

Comment:
Since we don't really need two nested `fold` that have the same behavior, what do you think of replacing with the following suggestion?

```suggestion
addNoticesHandler("post", "/paids/:eventId/disable", (req, res) => {
  pipe(
    req.params.eventId,
    O.fromNullable,
    O.fold(
      () => res.sendStatus(400),
      eventId => {
        NoticesDB.removeUserNotice(eventId);
        return res.sendStatus(hideReceiptResponseCode);
      }
    )
  );
});
```